_posts/2022-09-12-stargate-v2-beta.md

```diff
@@ -0,0 +1,77 @@
+* Cloud friendly. Being more service-oriented is of course also more cloud friendly. We’ve created a new “bridge” gRPC API to the coordinator node that makes it easy to create additional API services. Providing this HTTP-based API allows developers to extend the architecture more quickly than using traditional the CQL binary protocol and drivers. We’re also moving away from OSGi as our containerization standard, and instead making each service deployable as its own Kubernetes pod. Kubernetes is the de facto standard for cloud deployments, and we’re happy to align with that. This will also improve compatibility with the cloud native version of Cassandra, namely [K8ssandra](https://k8ssandra.io/).
+* Operator friendly. By leveraging Kubernetes, we can make each service independently deployable and scalable. This will give operators a lot more control over how they manage and scale Stargate.
+
+What hasn’t changed is our commitment to backward compatibility for all API endpoints. Additionally, Stargate v2 continues to support multiple Cassandra backends including Cassandra 3.11, Cassandra 4.0, and DataStax Enterprise 6.8. Stargate is also deployed as part of DataStax AstraDB, which provides important feedback on usage of Stargate at scale. To learn more about the Stargate v2 effort, check out our other blogs [announcing the project](https://stargate.io/2021/11/02/announcing-stargate-v2.html) and discussing the [design approach](https://stargate.io/2021/11/02/introducing-the-design-for-stargate-v2.html).
+
+## Why it’s time to release a Beta
+
+In our last update on Stargate v2, we [shared the progress](https://stargate.io/2022/01/06/reaching-the-first-milestone.html) on factoring the REST API out of the coordinator node as a separate microservice. Since that time we have created a new “Bridge” gRPC API distinct from the publicly available gRPC API and factored the GraphQL and Document API endpoints out of the coordinator into their own separate microservices. All three API services use the [Quarkus framework](https://quarkus.io/), which enables them to be very lightweight and leverage an up-to-date Java version - JDK 17.
+
+We are now getting close to a Release Candidate, and have a worthy Beta. This enables the open source community to start trying out Stargate v2 at near production quality in a couple of key areas:
+
+* Performance and scalability - we want to give you the opportunity to validate that performance continues to meet your expectations. It will also give you who use only certain APIs a chance to explore the new deployment and scaling possibilities, whereby you can deploy only the API services you actually need.
```

Comment:
```suggestion
* Performance and scalability - we want to give you the opportunity to validate that performance continues to meet your expectations. It will also allow you to explore the new deployment and scaling possibilities of a specific API , deploying only the API services you actually need.
```